openvariant/variant/variant.py

```diff
@@ -86,8 +85,12 @@ def _extract_header(file_path: str, original_header: list, annotation: Annotatio
         if ann_type == AnnotationTypes.MAPPING.value:
             mapping_fields.append((field, ann))
         else:
-            header_schema.update({field: AnnotationTypesProcess[ann_type].value(ann, original_header, file_path,
-                                                                                header_schema)})
+            class_name = ann_type
+            module_name = "openvariant.annotation.process"
+            ClassAnnotation = import_class_from_module(module_name, class_name)
+            instance = ClassAnnotation()
+
+            header_schema.update({field: instance(ann, original_header, file_path, header_schema)})
 
     for field, ann in mapping_fields:
         ann_type = ann[0]
```

Comment:
we are removing `AnnotationTypesProcess` but then calling it again line 97. This doesn't raise an error in the testing, but when running the linter (in my case I ran Ruff) it raised the error. Not sure why no test is failing. 

@dmartmillan

Reply:
Which error it raises? 

Probably, it comes because it is not imported, it not exist anymore. I guess we should change this line 97.

Reply:
yes, I was expecting the flake8 to fail though - then I checked and it's only reporting - not raising any error - should we change this behaviour? For example exit the workflow when FATAL errors are found. 

```sh
openvariant/variant/variant.py:97:38: F821 Undefined name `AnnotationTypesProcess`
   |
95 |     for field, ann in mapping_fields:
96 |         ann_type = ann[0]
97 |         header_schema.update({field: AnnotationTypesProcess[ann_type].value(ann, original_header, file_path,
   |                                      ^^^^^^^^^^^^^^^^^^^^^^ F821
98 |                                                                             header_schema)})
99 |     return header_schema, annotation.columns
   |
```